### PR TITLE
Feat: Uses `OutOfStock` component from FSUI

### PR DESCRIPTION
**Pull request by ArthurTriis1**
Signed-off-by: Arthur Andrade <arthurfelandrade@gmail.com>

This PR will replicate the changes made in this gatsby.store https://github.com/vtex-sites/gatsby.store/pull/34.

## What's the purpose of this pull request?

This PR is part of the Component Placement project. Using `OutOfStock`

## How does it work?

- Uses `OutOfStock` component from FSUI

## How to test it?

- Nothing should change visually on the component.

|PDP|
|-|
Screen Shot 2022-05-27 at 11 27 44

## Checklist

<em>You may erase this after checking them all ;)</em>

- [x] Added an entry in the `CHANGELOG.md` at the beginning of its due section. [The latest version should comes first](https://keepachangelog.com/en/1.0.0/#:~:text=The%20latest%20version%20comes%20first.).
- [x] Added the PR number with the PR link at the entry in the `CHANGELOG.md`. E.g., *New items in the `pull_request_template.md` ([#4](https://github.com/vtex-sites/nextjs.store/pull/4))*

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `EmptyState` component to be customized with a rounded `variant` prop (`default` | `rounded`) ([#11](https://github.com/vtex-sites/nextjs.store/pull/11)).
 - Uses camelCase classes when using CSS Modules ([#42](https://github.com/vtex-sites/nextjs.store/pull/42))
 - `ImageGallery` now uses native scroll instead of `useSlider` ([#6](https://github.com/vtex-sites/nextjs.store/pull/6))
+- `OutOfStock` component ([#72](https://github.com/vtex-sites/nextjs.store/pull/72))
```
Comment (danzanzini) [inline CHANGELOG.md:32]: Latest Entries should go on top

**Review by danzanzini — APPROVED** [review]

File: src/components/product/OutOfStock/OutOfStock.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs'
+import { SessionProvider } from '@faststore/sdk'
+
+
+import OutOfStock from '.'
+
+<Meta
+  title="Organisms/OutOfStock"
+  argTypes={{
+    buttonIcon: { control: false, },
```
Comment (eduardoformiga) [inline src/components/product/OutOfStock/OutOfStock.stories.mdx:10]: should we investigate how to change it afterward so that we can select other icons?

**Review by eduardoformiga — APPROVED** [review]

File: src/components/product/OutOfStock/OutOfStock.stories.mdx (hunk 2 above)
Comment (hellofanny) [reply src/components/product/OutOfStock/OutOfStock.stories.mdx:10]: Hmm, but we will need to provide those icons (list some icons example to choose?) 🤔

**Review by hellofanny — APPROVED** [review]
LGTM! 🚀 🚀 🚀

File: src/components/product/OutOfStock/OutOfStock.stories.mdx (hunk 2 above)
Comment (eduardoformiga) [reply src/components/product/OutOfStock/OutOfStock.stories.mdx:10]: I was thinking outloud. Not sure about this idea hahah

Comment (hellofanny) [reply src/components/product/OutOfStock/OutOfStock.stories.mdx:10]: Let's save this idea for the future, then 😉

**Pull request merged**